src/schemathesis/_hypothesis.py

```diff
@@ -33,15 +36,22 @@ def create_test(
 ) -> Callable:
     """Create a Hypothesis test."""
     hook_dispatcher = getattr(test, "_schemathesis_hooks", None)
-    strategy = endpoint.as_strategy(hooks=hook_dispatcher)
+    from .runner.impl.core import Feedback  # pylint: disable=import-outside-toplevel
+
+    feedback = Feedback("links", endpoint)
+    strategy = get_case_strategy(endpoint, hook_dispatcher, feedback=feedback)
     wrapped_test = hypothesis.given(case=strategy)(test)
     if seed is not None:
         wrapped_test = hypothesis.seed(seed)(wrapped_test)
     if asyncio.iscoroutinefunction(test):
         wrapped_test.hypothesis.inner_test = make_async_test(test)  # type: ignore
     setup_default_deadline(wrapped_test)
     if settings is not None:
-        wrapped_test = settings(wrapped_test)
+        try:
+            wrapped_test = settings(wrapped_test)
+        except:  # TODO. fix re-wrapping
```

Comment:
We need to store the original test because another endpoint will have other examples, etc. After that, this try/except won't be needed